news/4096.core.rst

```diff
@@ -0,0 +1 @@
+Now uses PEP 552 for the `.pyc` files in `base_library.zip` when using Python 3.7.
```

Comment:
This is not quite correct, since PEP 552 is followed since #3626 already. This is more like "uses hash based …"

You can also write `:pep:`552`` which will create a reference to the resp. PEP.

Reply:
Done